### ISSUE TITLE
GSE104276 - Prefrontal cortex development

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/hca-ebi-wrangler-central#61]
**Primary Wrangler:**
Enrique
**Secondary Wrangler:**
@ami-day 

**Associated files:**

Google Drive: https://drive.google.com/drive/folders/1xx36X2lMzux1nWFmrSxhHZA6VYwGkZZK

**Key Events**
- [x] Assign the Primary Wrangler
- [x] Fill spreadsheet (to a sufficient standard to submit dataset) 
- [x] Get data into s3 (to a sufficient standard to submit dataset)
- [x] Curate metadata spreadsheet with ontologies
- [x] Upload sheet to validate metadata
- [x] Run graph validator
- [x] Check linking
- [x] Validate data files
- [x] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [ ] Get final approval of submission including metadata spreadsheet and data files
- [x] Submit to production and inform ingest-devs the project is ready for archiving
- [ ] Email UX to inform that they're ready to send survey

**Please track the below as well as the key events:**
1. Track date first spreadsheet received and final spreadsheet sent by editing ticket to include
date next to event.
1. Track spreadsheet iterations by placing asterisks next to receive spreadsheet event.
1. Track any metadata issues/tickets made for dataset with a bulleted list of links under
received spreadsheet event. Links should be to the ticket in the metadata repo.

**Comment by ESapenaVentura** [comment]
**UPDATE**:

- Uploaded metadata spreadsheet
   - Everything is fine aside from project/study accesions
- Ran ingest-graph-validator tool:
   - No errors with pypi version of the graph-validator

**Comment by ESapenaVentura** [comment]
Data was uploaded to the upload area. Also testing the files/files metadata in ingest under submission [961412fc-eb29-4c35-a070-e4b24182d7a8](https://ui.ingest.archive.data.humancellatlas.org/submissions/detail?uuid=961412fc-eb29-4c35-a070-e4b24182d7a8&project=ca27b835-4fbe-45a9-98d9-27440f18cbef)

**Comment by ami-day** [comment]
Ami Review:

In general it looks very good. I've suggested a couple of small changes that I have not addressed directly in the sheet that you might want to change:

- In the contributor's tab, the full address is in the Institute column. Should we split this up into the separate columns for institute, department, street address, country in order to be consistent with other datasets? 

- In the donor organism descriptions, there is the following "Fetal donor for pre-frontal cortext tissue from 23th Post-conception week" (twice). This should be "23rd".

- Just an observation: single cells were picked with mouth pipette :O

This looks like a really good dataset to add, given it is primary normal brain and difficult kind of material for researchers to get hold of.

**Comment by ESapenaVentura** [comment]
Thanks for the comments @ami-day ! I have addressed the addresses' issue (Not redundant at all!) as well as the spelling errors.

I'll proceed to re-upload the project! Since it's both non-project and project metadata changes I think it's worth deleting the previous submission and creating a new one!

**Comment by ESapenaVentura** [comment]
A new project/submission has been created and files are being validated:
https://ui.ingest.archive.data.humancellatlas.org/submissions/detail?uuid=53f376be-5905-4067-8b06-6a4cc6756e2d&project=8714dd41-ee78-42a5-829c-7478ba74813c

**Comment by ESapenaVentura** [comment]
The dataset had linking errors due to donor/specimen sharing names. New submission/project:

https://ui.ingest.archive.data.humancellatlas.org/submissions/detail?uuid=a7215b2d-429f-4099-96d8-bcfc9c37e271&project=a815c84b-8999-433f-958e-422c0720e00d

**Comment by ESapenaVentura** [comment]
Dataset is waiting for export to DCP2 - No more wrangling/tracking needed for now. Closing the ticket!

**Comment by mshadbolt** [comment]
Is this worth reopening to track for SCEA brokering?

**Comment by ESapenaVentura** [comment]
Meh, this is the dataset that has the multiplexed SS2 files... I haven't heard from the contributor yet

**Comment by mshadbolt** [comment]
ohhh right, I guess reopening and pushing status back to exporting then?

**Comment by Wkt8** [comment]
According to the Ingest API and the staging area google buckets, only the metadata has been exported for this project - none of the raw data files. 

This project is also not in the DCP. 

Project here: https://contribute.data.humancellatlas.org/projects/detail?uuid=a815c84b-8999-433f-958e-422c0720e00d

**Comment by Wkt8** [comment]
@ESapenaVentura and @ke4 to work on this (state tracker) and re-export to the DCP.

**Comment by ami-day** [comment]
This can be converted to SCEA. Assigning it E-HCAD-id:  E-HCAD-id-45

**Comment by ami-day** [comment]
Manually curated the files and uploaded them to Gitlab. Merge request: https://gitlab.ebi.ac.uk/ebi-gene-expression/scxa-metadata/-/merge_requests/293

Note: the E-HCAD was reassigned to **E-HCAD-46.**

**Comment by prabh-t** [comment]
Enrique is not able to unblock and release this dataset.

**Comment by ESapenaVentura** [comment]
Exported - sent import form request

**Comment by ESapenaVentura** [comment]
Missing matrix files - Will include them in following release

**Comment by ofanobilbao** [comment]
@ESapenaVentura  Are you planning on adding this to Release 17? If so, please, add label. Thanks

**Comment by Wkt8** [comment]
Enrique is almost done. This is great! The linking changes have been made programatically and the new processes have been created manually via the UI(?). 

@Wkt8 to test linking to pre-existing submissions via new submissions.

**Comment by ESapenaVentura** [comment]
The new submission with the matrix files is currently being exported:
- https://contribute.data.humancellatlas.org/submissions/detail?uuid=f36dafb0-4368-4b87-80bf-58743ae16555&project=34da2c5f-8011-48af-a7fd-ad2f56ec10f4

Once that finishes, however, I need to update it quickly as I realised it's missing the "source" field and an ontology term in one of the files `content_description` fields

**Comment by Wkt8** [comment]
Enrique to export again!

**Comment by ESapenaVentura** [comment]
https://contribute.data.humancellatlas.org/submissions/detail?uuid=2ad60ae7-9274-4eaa-b22a-2b345d5201b0&project=a815c84b-8999-433f-958e-422c0720e00d

Is stuck in exporting

Also, spreadsheet is not being generated

**Comment by ESapenaVentura** [comment]
Stuck in exporting - It's a weird case, no export jobs have been sent https://app.zenhub.com/workspaces/operations-5fa2d8f2df78bb000f7fb2b5/issues/ebi-ait/hca-ebi-wrangler-central/869

**Comment by MightyAx** [comment]
All of the jobs for this submission have then exported successfully!

**Comment by ESapenaVentura** [comment]
Updated the file metadata (Added a field), re-validated and hit re-export

**Comment by ESapenaVentura** [comment]
Sent import form

**Comment by ESapenaVentura** [comment]
Sent import form
